ScnSoft-Education/LeetCodeTraining/LongestPalindromicSubstring4/Longest Palindromic Substring.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace LeetCodeTraining.LongestPalindromicSubstring4
+{
+    public class Solution
+    {
+        public string LongestPalindrome(string s)
+        {
+            if (string.IsNullOrEmpty(s))
+            {
+                return "";
+            }
+
+            string result = "";
+
+            for (int i = 0; i < s.Length; i++)
+            {
+                for (int j = i + 1; j <= s.Length; j++)
+                {
+                    string substr = s.Substring(i, j - i);
```

Comment:
what does 'substr' mean? please rename it according to the sense

Reply:
renaming variables